src/ui/osx/TogglDesktop/test2/UnsupportedNotice.h

```diff
@@ -0,0 +1,21 @@
+//
+//  UnsupportedNotice.h
+//  TogglDesktop
+//
+//  Created by Indrek Vändrik on 23/11/2018.
+//  Copyright © 2018 Alari. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface UnsupportedNotice : NSAlert
```

Comment:
IMO, it makes no sense that `UnsupportedNotice` is subclass from `NSAlert`, which is UI components.
The reason is that we're trying to support Singleton for a view, which is bad practice.

I suggest that `UnsupportedNotice` should be a true model, which subclass from `NSObject`.

Reply:
I don't understand the reasoning behind this. UnsupportedNotice is a simple Alert time popup for the user. Why would we need to create a true model out of it? It would make sense to create a model if we would decide to use it as a general notice that can be used with different scenarios. Currently, it is not the case and there is no reason to treat it that way.

Reply:
I understand your reason. 

My concern is related to the change in the future. I predict that we might use this `UnsupportedNotice` class for many extensions soon. As a result, it's better be a true model, then next teammate, who are in charge, easily follow to extend new feature.

However, for now, it's okay to keep it as it is.